src/raylib/rlgl.mli

```diff
@@ -0,0 +1,300 @@
+val ortho : float -> float -> float -> float -> float -> float -> unit
+
+val viewport : int -> int -> int -> int -> unit
+
+val rl_begin : int -> unit
+
+val rl_end : unit -> unit
+
+val vertex2i : int -> int -> unit
+
+val vertex2f : float -> float -> unit
+
+val vertex3f : float -> float -> float -> unit
+
+val tex_coord2f : float -> float -> unit
+
+val normal3f : float -> float -> float -> unit
+
+val color4ub :
+  Unsigned.uchar -> Unsigned.uchar -> Unsigned.uchar -> Unsigned.uchar -> unit
+
+val color3f : float -> float -> float -> unit
+
+val color4f : float -> float -> float -> float -> unit
+
+val enable_vertex_array : Unsigned.uint -> bool
+
+val disable_vertex_array : unit -> unit
+
+val enable_vertex_buffer : Unsigned.uint -> unit
+
+val disable_vertex_buffer : unit -> unit
+
+val enable_vertex_buffer_element : Unsigned.uint -> unit
+
+val disable_vertex_buffer_element : unit -> unit
+
+val enable_vertex_attribute : Unsigned.uint -> unit
+
+val disable_vertex_attribute : Unsigned.uint -> unit
+
+val active_texture_slot : int -> unit
+
+val enable_texture : Unsigned.uint -> unit
+
+val disable_texture : unit -> unit
+
+val enable_texture_cubemap : Unsigned.uint -> unit
+
+val disable_texture_cubemap : unit -> unit
+
+val texture_parameters : Unsigned.uint -> int -> int -> unit
+
+val enable_shader : Unsigned.uint -> unit
+
+val disable_shader : unit -> unit
+
+val enable_framebuffer : Unsigned.uint -> unit
+
+val disable_framebuffer : unit -> unit
+
+val enable_depth_test : unit -> unit
+
+val disable_depth_test : unit -> unit
+
+val enable_depth_mask : unit -> unit
+
+val disable_depth_mask : unit -> unit
+
+val enable_backface_culling : unit -> unit
+
+val disable_backface_culling : unit -> unit
+
+val enable_scissor_test : unit -> unit
+
+val disable_scissor_test : unit -> unit
+
+val scissor : int -> int -> int -> int -> unit
+
+val enable_wire_mode : unit -> unit
+
+val disable_wire_mode : unit -> unit
+
+val set_line_width : float -> unit
+
+val get_line_width : unit -> float
+
+val enable_smooth_lines : unit -> unit
+
+val disable_smooth_lines : unit -> unit
+
+val enable_stereo_render : unit -> unit
+
+val disable_stereo_render : unit -> unit
+
+val is_stereo_render_enabled : unit -> bool
+
+val clear_color :
+  Unsigned.uchar -> Unsigned.uchar -> Unsigned.uchar -> Unsigned.uchar -> unit
+
+val clear_screen_buffers : unit -> unit
+
+val check_errors : unit -> unit
+
+val set_blend_mode : int -> unit
+
+val set_blend_factors : int -> int -> int -> unit
+
+val rlgl_init : int -> int -> unit
+
+val rlgl_close : unit -> unit
+
+val load_extensions : unit Ctypes_static.ptr -> unit
+
+val get_version : unit -> int
+
+val get_framebuffer_width : unit -> int
+
+val get_framebuffer_height : unit -> int
+
+val get_texture_id_default : unit -> Unsigned.uint
+
+val get_shader_id_default : unit -> Unsigned.uint
+
+val load_render_batch :
+  int ->
+  int ->
+  Raylib_generated_types__.Rlgl_generated_types.RlRenderBatch.t Ctypes.structure
```

Comment:
Instead of having these long generated types, I would create separate modules for each rlgl type, like the main raylib does for eg `Vector2`.
We still can merge this and polish it later